cloud-aws/src/test/java/com/sequenceiq/cloudbreak/cloud/aws/view/AwsRdsDbParameterGroupViewTest.java

```diff
@@ -131,4 +131,34 @@ void getDBParameterGroupFamilyTestWhenPgSqlAndValidVersion(String testCaseName,
         assertThat(underTest.getDBParameterGroupFamily()).isEqualTo(familyExpected);
     }
 
+    @Test
+    void templateAndDefaultPostgresVersionIsMatching() throws IOException {
+        assertThat(AwsRdsDbParameterGroupView.DEFAULT_FAMILY_VERSION).isEqualTo(getDefaultEngineVersionFromTemplate());
+    }
+
+    private String getDefaultEngineVersionFromTemplate() throws IOException {
+        String template = getTemplate();
+        Pattern defaultEngineVersion = Pattern.compile(".*EngineVersionParameter.*Default\": \"(\\d*)\".*");
```

Comment:
Not a big deal but this could be also a constant.

Reply:
ok